src/App.js

```diff
@@ -1,12 +1,19 @@
-import React from "react";
+import React, { useState } from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
 function App() {
+  const [newTodo, setNewTodo] = React.useState(" ");
```

Comment:
this is a very small nitpick, but i'm not sure if a string with one space in it is technically the same as an empty string for some purposes. Maybe safer to delete the empty space:

```suggestion
  const [newTodo, setNewTodo] = React.useState("");
```